linter/linter.go

```diff
@@ -260,10 +274,54 @@ func (l *Linter) lintVCL(vcl *ast.VCL, ctx *context.Context) types.Type {
 	return types.NeverType
 }
 
+func (l *Linter) resolveIncludeStatements(statements []ast.Statement, ctx *context.Context) []ast.Statement {
+	var resolved []ast.Statement
+
+	for _, stmt := range statements {
+		include, ok := stmt.(*ast.IncludeStatement)
+		if !ok {
+			resolved = append(resolved, stmt)
+			continue
+		}
+
+		// Check snippet inclusion
+		if strings.HasPrefix(include.Module.Value, "snippet::") {
+			// TODO: implement fastly managed snippet inclusion
```

Comment:
We should implement for solving https://github.com/ysugimoto/falco/issues/118 on another PR